src/tests/reader.rs

```diff
@@ -113,10 +113,7 @@ fn two_dimension_on_a_line() {
 
     // if we can't look into enough nodes we find some random points
     let ret = reader.nns(5).search_k(NonZeroUsize::new(1).unwrap()).by_item(&rtxn, 1).unwrap();
-    insta::assert_snapshot!(NnsRes(ret), @r###"
-    id(48): distance(47)
-    id(92): distance(91)
-    "###);
+    insta::assert_snapshot!(NnsRes(ret), @"id(63): distance(62)");
```

Comment:
Can you explain again why it changed?

Reply:
Because this specific test is telling arroy to stop searching after it find its first leaf, so the results are completely random and may change every time we change the database (it's written in a comment above the assertion)